reduction/lr_reduction/template.py

```diff
@@ -142,13 +142,14 @@ def process_from_template(run_number, template_path, q_summing=False, normalize=
     return process_from_template_ws(ws_sc, template_path, q_summing=q_summing,
                                     tof_weighted=tof_weighted, bck_in_q=bck_in_q,
                                     clean=clean, info=info, normalize=normalize,
-                                    functional_background=functional_background)
+                                    functional_background=functional_background,
+                                    dead_time=dead_time)
 
 
 def process_from_template_ws(ws_sc, template_data, q_summing=False,
                              tof_weighted=False, bck_in_q=False, clean=False,
                              info=False, normalize=True, theta_value=None, ws_db=None,
-                             functional_background=False):
+                             functional_background=False, dead_time=False):
```

Comment:
is the optional argument `paralyzable` of `EventReflectivity.__init__` left out here on purpose?